src/components/todo-item/todo-item.component.scss

```diff
@@ -17,6 +17,17 @@
     }
   }
 
+  .block-picker {
```

Comment:
i do not like that name. color-picker will be better.

and if you trying to access BlockPicker (component), this is not the way.

You should wrap it with additional div or span or whatever, and add className


.XYZ in CSS means element with class XYZ